pages/contracts.tsx

```diff
@@ -132,8 +133,9 @@ const ContractList = () => {
                     </tr>
                   ))
                 ) : data.meta.totalPage ? (
-                  data.contracts.map(c => (
+                  data.contracts.map((c, idx) => (
                     <tr key={c.id} title={c.address}>
+                      <td>{idx + 1}</td>
```

Comment:
The index would repeat if `page number` is not considered

Reply:
ok. I got it.

Reply:
@Keith-CY Hold a minute. we are now using graphql requests to get data, and we paginating by the cursor. So, how can we get the current page number in the frontend?

Reply:
Good question, it seems extremely hard to set a rank, although ranked tokens/contracts by time are easy, sorted by balance or holders, etc. are difficult, any idea or advice from @Naupio